pkg/core/network.go

```diff
@@ -378,6 +412,45 @@ func (n *NetworkCommand) NeedApplyTC() bool {
 	}
 }
 
+func (n *NetworkCommand) ToChain(ipset string) ([]*pb.Chain, error) {
```

Comment:
I think ToChain is not a good name to explain what the func do.

Reply:
Do you have any suggestions?

Reply:
Maybe NetworkPartitionChains?

Reply:
OK，I update to `PartitionChains` without `Network`, because the object's type is `NetworkCommand `